string job_id           # Unique ID of the Job.
string priority         # Priority of this job; 1=LOW, 2=MEDIUM, 3=HIGH, 4=CRITICAL.
uint8 task_count        # Number of tasks the Job has.
string keyword          # Order keyword, e.g. TRANSPORT or MOVE